pandas/tests/arrays/test_array.py

```diff
@@ -391,61 +390,3 @@ def test_array_not_registered(registry_without_decimal):
     result = pd.array(data, dtype=DecimalDtype)
     expected = DecimalArray._from_sequence(data)
     tm.assert_equal(result, expected)
-
-
-class TestArrayAnalytics:
-    def test_searchsorted(self, string_dtype):
-        arr = pd.array(["a", "b", "c"], dtype=string_dtype)
-
-        result = arr.searchsorted("a", side="left")
-        assert is_scalar(result)
-        assert result == 0
-
-        result = arr.searchsorted("a", side="right")
-        assert is_scalar(result)
-        assert result == 1
-
-    def test_searchsorted_numeric_dtypes_scalar(self, any_real_numpy_dtype):
-        arr = pd.array([1, 3, 90], dtype=any_real_numpy_dtype)
-        result = arr.searchsorted(30)
-        assert is_scalar(result)
-        assert result == 2
-
-        result = arr.searchsorted([30])
-        expected = np.array([2], dtype=np.intp)
-        tm.assert_numpy_array_equal(result, expected)
-
-    def test_searchsorted_numeric_dtypes_vector(self, any_real_numpy_dtype):
-        arr = pd.array([1, 3, 90], dtype=any_real_numpy_dtype)
-        result = arr.searchsorted([2, 30])
-        expected = np.array([1, 2], dtype=np.intp)
-        tm.assert_numpy_array_equal(result, expected)
-
-    @pytest.mark.parametrize(
-        "arr, val",
-        [
-            [
-                pd.date_range("20120101", periods=10, freq="2D"),
-                pd.Timestamp("20120102"),
-            ],
-            [
-                pd.date_range("20120101", periods=10, freq="2D", tz="Asia/Hong_Kong"),
-                pd.Timestamp("20120102", tz="Asia/Hong_Kong"),
-            ],
-            [
-                pd.timedelta_range(start="1 day", end="10 days", periods=10),
-                pd.Timedelta("2 days"),
-            ],
-        ],
-    )
-    def test_search_sorted_datetime64_scalar(self, arr, val):
```

Comment:
This get dropped?

Reply:
yes, redundant with existing tests in tests.arrays.test_datetimelike